src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
> I am sure that you are even more disappointed by that decision.
You don't even know the half, it was a stab to the heart and the feels were real. I wholeheartedly agree with you and that's why I am not budging when it comes to the development environment, that will all be TypeScript and unless they modify the build environment to support watch JavaScript files then it won't actually watch for it....I am extremely tempted to not support JavaScript files _until_ version two or three that way people get used to the APIs and give TypeScript a try. Since, as you said, if they go the JavaScript route they are loosing the typed nature and if they use an IDE like VSCode they loose autocomplete along with code documentation which they then have to go manually searching through code and documentation.

> One of the major USPs of Rocket.Chat compared to others is it's availability on premise with enterprise features for companies. I'd not install a single Rocketlet if I was unsure what it can do.
Due to the large amount of work it would take to get this, I'm willing to put it on the backlog for the first alpha version and try to support it in the second alpha or at least get basic support for it before beta version.